src/test/java/org/opensearch/security/system_indices/SystemIndicesTests.java

```diff
@@ -58,10 +61,16 @@ public class SystemIndicesTests extends SingleClusterTest {
         allAccessUser
     );
 
+    private static final String extensionUser = "extensions_user";
```

Comment:
Don't name these `extension`, it isn't needed.  this feature isn't limited to extensions

Reply:
i used extensions for the tests cause is a concrete scenario. What you think i should name them? systemIndexUser?